### PR TITLE
Add --unformat (-u) argument

### PR DESCRIPTION
**Pull request by thepwrtank18**
This argument removes all formatting from the output, so instead of this:
```
>xpkey -n 5
XXXXX-XXXXX-XXXXX-XXXXX-XXXXX

XXXXX-XXXXX-XXXXX-XXXXX-XXXXX

XXXXX-XXXXX-XXXXX-XXXXX-XXXXX

XXXXX-XXXXX-XXXXX-XXXXX-XXXXX

XXXXX-XXXXX-XXXXX-XXXXX-XXXXX

Success count: 5/5

```

```
>xpkey -i XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX
Confirmation ID: XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX

```

You can use this:
```
>xpkey -n 5 -u
XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
XXXXX-XXXXX-XXXXX-XXXXX-XXXXX
```

```
>xpkey -i XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX -u
XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX
```
It's a way for other programs to "get in, get keys, get out" without having to modify the output it got.

### CODE REVIEW

**Comment by WitherOrNot** [comment]
Perhaps change to `-r/--raw`, since `-u/--upgrade` should be for specifying upgrade bit

File: src/cli.cpp [hunk 1]
```diff
@@ -218,16 +222,26 @@ void CLI::printID(DWORD *pid)
...
+            spk.substr(20, 5));
+    }
+    else {
+        fmt::print("{}-{}-{}-{}-{}\n",
+            spk.substr(0, 5),
+            spk.substr(5, 5),
+            spk.substr(10, 5),
+            spk.substr(15, 5),
+            spk.substr(20, 5));
+    }
```
Comment (WitherOrNot) [inline src/cli.cpp:244]: Clean up this if statement
```suggestion
	fmt::print("{}-{}-{}-{}-{}",
		spk.substr(0, 5),
		spk.substr(5, 5),
		spk.substr(10, 5),
		spk.substr(15, 5),
		spk.substr(20, 5));
	
	if (!options.unformatted) {
		fmt::print("\n");
	}
```

File: src/cli.cpp [hunk 2]
```diff
@@ -304,14 +318,16 @@ int CLI::BINK1998() {
 
     for (int i = 0; i < this->total; i++) {
         BINK1998::Generate(this->eCurve, this->genPoint, this->genOrder, this->privateKey, nRaw, bUpgrade, this->pKey);
-        CLI::printKey(this->pKey);
-        fmt::print("\n");
-
+        CLI::printKey(this->pKey, this->options);
+        if (this->options.unformatted == false || i < this->total-1) {
```
Comment (WitherOrNot) [inline src/cli.cpp:322]:
```suggestion
        if (!this->options.unformatted || i < this->total - 1) {
```

File: src/cli.cpp [hunk 3]
```diff
@@ -304,14 +318,16 @@ int CLI::BINK1998() {
...
+            fmt::print("\n");
+        }
         // verify the key
         this->count += BINK1998::Verify(this->eCurve, this->genPoint, this->pubPoint, this->pKey);
     }
-
-    fmt::print("Success count: {}/{}\n", this->count, this->total);
+    if (this->options.unformatted == false) {
```
Comment (WitherOrNot) [inline src/cli.cpp:328]:
```suggestion
    if (!this->options.unformatted) {
```

File: src/cli.h [hunk 4]
```diff
@@ -43,11 +43,11 @@ class CLI {
     static int parseCommandLine(int argc, char* argv[], Options *options);
     static int validateCommandLine(Options* options, char *argv[], json *keys);
     static void printID(DWORD *pid);
-    static void printKey(char *pk);
+    static void printKey(char *pk, Options options);
 
     int BINK1998();
     int BINK2002();
-    int ConfirmationID();
+    int ConfirmationID(Options options);
```
Comment (WitherOrNot) [inline src/cli.h:50]: `options` is a global variable, should not be specified as an argument

File: src/main.cpp [hunk 5]
```diff
@@ -49,7 +49,7 @@ int main(int argc, char *argv[]) {
             return run->BINK2002();
 
         case MODE_CONFIRMATION_ID:
-            return run->ConfirmationID();
+            return run->ConfirmationID(options);
```
Comment (WitherOrNot) [inline src/main.cpp:52]: same here, `options` shouldnt be an arg

**Review by WitherOrNot — CHANGES_REQUESTED** [review]
Some formatting fixes are needed, but the logic looks good

**Review by WitherOrNot — APPROVED** [review]

**Comment by thepwrtank18** [comment]
Closing this since Git is being stupid

**Pull request closed without merging**